src/module-info.java

```diff
@@ -1,6 +1,7 @@
 module RpgInventorySystem {
-	requires javafx.controls;
+	/*requires javafx.controls;*/
```

Comment:
Why was this removed? It prevents running main and javafx will be needed by our UI